### PR TITLE
docs: prolog documentation

### LINKED ISSUE #78: Adicionar documentação de Prolog

### PR DESCRIPTION
**Pull request by diego-aquino**
### Docs
- Added the documentation about the Prolog project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +48,46 @@ Para utilizar a versão funcional desse projeto, escrita em [Haskell](https://ww
    cabal run
    ```
 
-Para uma melhor experiência, recomenda-se ajustar o tamanho do terminal para as seguintes dimensões>
+# Projeto funcional (Prolog)
```
Comment (Murilo-Gruppi) [inline README.md]: The title should be `Projeto Lógico`.

Comment (diego-aquino) [reply README.md]: Thanks! My bad.

**Review by Murilo-Gruppi — APPROVED** [review]
Well Done!!

**Review by edugabriel12 — APPROVED** [review]
Nice!

**Review by augustonz — APPROVED** [review]

**Pull request merged**